.dependabot/config.yml

```diff
@@ -0,0 +1,10 @@
+version: 1
+update_configs:
+  - package_manager: "python"
+    directory: "/"
+    update_schedule: "weekly"
+    default_labels:
+      - "dependencies"
+    default_reviewers:
+      - "Zlopez"
+      - "jeremycline"
```

Comment:
`default_reviewers` sourced from members of `release-monitoring`.
<img width="1440" alt="Screen Shot 2019-10-04 at 08 47 51" src="https://user-images.githubusercontent.com/1557529/66171587-aa133300-e683-11e9-97d0-9f64f8188817.png">